<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistPage</class>
 <widget class="QWizardPage" name="PlaylistPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/icons/images/tano.png</normaloff>:/icons/images/tano.png</iconset>
  </property>
  <property name="title">
   <string>Select Playlist</string>
  </property>
  <property name="subTitle">
   <string>Please, select your IP TV provider or browse for custom playlist.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="defaultPlaylistBox">
     <property name="title">
      <string>Default Channel list</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QRadioButton" name="presetPlaylistRadio">
        <property name="text">
         <string>Preset</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="browseLayout">
        <item>
         <widget class="QLineEdit" name="playlistLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="browsePlaylistButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetPlaylistButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="customPlaylistRadio">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="presetsBox">
     <property name="title">
      <string>Presets</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QRadioButton" name="radioSiol2">
        <property name="text">
         <string notr="true">SiOL - MPEG-2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSiol4">
        <property name="text">
         <string notr="true">SiOL - MPEG-4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioT2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string notr="true">T-2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioTus">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string notr="true">Tu≈° Telekom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>customPlaylistRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetsBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPlaylistRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>playlistLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPlaylistRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>browsePlaylistButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPlaylistRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>resetPlaylistButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
